src/utils/client.ts

```diff
@@ -910,6 +910,110 @@ export default class Client {
       }
     });
   }
+
+  // Telesign
+  async telesignSilentVerification(phoneNumber: string) {
+    const { userId } = initInfo();
+    const initialUrl = `${API_URL}telesign/silent/initial/`;
+    const options = {
+      headers: { 'Content-Type': 'application/json' },
+      method: 'POST',
+      body: JSON.stringify({
+        phone_number: phoneNumber,
+        form_key: this.formKey,
+        fuser_key: userId
+      })
+    };
+    try {
+      const initialResponse = await this._fetch(initialUrl, options, false);
+      if (initialResponse) {
+        const { verification, reference_id, status } = await initialResponse.json();
+        if (!status) return status;
+
+        // Kick off process to establish session with carrier fron client side
+        const { verification_url, method, expected_response_code, post_body, query_string_params } = verification;
```

Comment:
snake case parameters should be set to camelCase variables, e.g. const { verification_url: verifUrl } = verification;

shouldn't use snake case variables in JS